CHANGELOG.md

```diff
@@ -23,6 +23,23 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## Develop
+### Changed
+- Changed unit test expected values due to update to PVWatts v8, which slightly changed expected PV production factors.
+- Changed **fuel_avail_gal** default to 1e9 for on-grid scenarios (same as off-grid)
+### Fixed
+- Corrected `Generator` **installed_cost_per_kw** from 500 to 650 if **only_runs_during_grid_outage** is _true_ or 800 if _false_
+
+## v0.32.4
```

Comment:
This stuff under 0.32.4 hasn't been merged into master and released yet right? Need to combine with what's under Develop above.

Reply:
done